### ISSUE TITLE
Host repo not cloning

### ISSUE THREAD

**Issue by orcutt989** [nektos/act#1236]
### Bug report info

```plain text
act --bug-report
act version:            0.2.28
GOOS:                   windows
GOARCH:                 amd64
NumCPU:                 20
Docker host:            DOCKER_HOST environment variable is unset/empty.
Sockets found:
        \\.\pipe\docker_engine
Config files:
        C:\Users\xxx\.actrc:
                -P ubuntu-latest=ghcr.io/catthehacker/ubuntu:act-latest
                --secret-file my.secrets
Build info:
        Go version:            go1.18.3
        Module path:           github.com/nektos/act
        Main version:          (devel)
        Main path:             github.com/nektos/act
        Main checksum:
                -ldflags:             -s -w -X main.version=0.2.28 -X main.commit=de37f750773575651a91363eb6b1ce97be964845 -X main.date=2022-06-21T14:07:09Z -X main.builtBy=goreleaser
                CGO_ENABLED:          0
                GOARCH:               amd64
                GOOS:                 windows
                GOAMD64:              v1
                vcs:                  git
                vcs.revision:         de37f750773575651a91363eb6b1ce97be964845
                vcs.time:             2022-06-21T13:52:21Z
                vcs.modified:         false
Docker Engine:
        Engine version:        20.10.16
        Engine runtime:        runc
        Cgroup version:        1
        Cgroup driver:         cgroupfs
        Storage driver:        overlay2
        Registry URI:          https://index.docker.io/v1/
        OS:                    Docker Desktop
        OS type:               linux
        OS version:
        OS arch:               x86_64
        OS kernel:             5.10.16.3-microsoft-standard-WSL2
        OS CPU:                1
        OS memory:             1983 MB
        Security options:
                name=seccomp,profile=default
```

### Command used with act

```sh
act -P ubuntu-latest=ghcr.io/catthehacker/ubuntu:act-latest --secret-file my.secrets
```

### Describe issue

With the following action-

```yaml
    # Steps represent a sequence of tasks that will be executed as part of the job
    steps:
      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
      - name: ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️ Check out this repo ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️
        uses: actions/checkout@v3
        
      - run:
          ls -lah &&
          sleep infinity
```

I expect the files `assets` and `paris-syndrome.gbsproj` to be available to `act` but the directory seems to be empty...

They are not in the gitignore.

```
PS C:\Users\xxx\src\paris-syndrome-gb> ls


    Directory: C:\Users\xxx\src\paris-syndrome-gb


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----         6/30/2022  12:59 AM                .git
d-----         6/24/2022   8:54 PM                .github
d-----         6/24/2022   9:31 PM                assets
-a----         6/30/2022  12:55 AM            717 .gitignore
-a----         6/29/2022  11:22 PM            109 my.secrets
-a----         6/24/2022  10:41 PM          47211 paris-syndrome.gbsproj
PS C:\Users\xxx\src\paris-syndrome-gb> act
[CI/build] 🚀  Start image=ghcr.io/catthehacker/ubuntu:act-latest
[CI/build]   🐳  docker pull image=ghcr.io/catthehacker/ubuntu:act-latest platform= username= forcePull=false
[CI/build]   🐳  docker create image=ghcr.io/catthehacker/ubuntu:act-latest platform= entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[CI/build]   🐳  docker run image=ghcr.io/catthehacker/ubuntu:act-latest platform= entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[CI/build]   🐳  docker exec cmd=[chown -R 0:0 /mnt/c/Users/xxx/src/paris-syndrome-gb] user=0 workdir=
[CI/build]   ☁  git clone 'https://github.com/c-hive/gha-yarn-cache' # ref=v2
[CI/build]   ☁  git clone 'https://github.com/Borales/actions-yarn' # ref=v3.0.0
[CI/build]   ☁  git clone 'https://github.com/actions/setup-node' # ref=v3
[CI/build]   ☁  git clone 'https://github.com/Borales/actions-yarn' # ref=v3.0.0
[CI/build]   ☁  git clone 'https://github.com/Borales/actions-yarn' # ref=v3.0.0
[CI/build]   ☁  git clone 'https://github.com/KikimoraGames/itch-publish' # ref=v0.0.3
[CI/build] ⭐ Run Main ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️ Check out this repo ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️
[CI/build]   🐳  docker cp src=C:\Users\xxx\src\paris-syndrome-gb\. dst=\mnt\c\Users\xxx\src\paris-syndrome-gb
[CI/build] close C:\Users\xxx\AppData\Local\Temp\act3952977024: file already closed
[CI/build]   🐳  docker exec cmd=[chown -R 0:0 \mnt\c\Users\xxx\src\paris-syndrome-gb] user=0 workdir=
[CI/build]   ✅  Success - Main ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️ Check out this repo ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️
[CI/build] ⭐ Run Main echo $(pwd) && ls -lah && sleep infinity
[CI/build]   🐳  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/1] user= workdir=
| /mnt/c/Users/xxx/src/paris-syndrome-gb
| total 8.0K
| drwxr-xr-x 2 root root 4.0K Jun 30 17:33 .
| drwxr-xr-x 3 root root 4.0K Jun 30 17:33 ..
```

### Workflow content

```yml
# This is a basic workflow to help you get started with Actions

name: CI

# Controls when the workflow will run
on:
  # Triggers the workflow on push or pull request events but only for the "master" branch
  push:
    branches: [ "master" ]
  pull_request:
    branches: [ "master" ]
  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:

env:
  ITCH_USERNAME: honestpizza
  ITCH_GAME_ID: paris-syndrome

# A workflow run is made up of one or more jobs that can run sequentially or in parallel
jobs:
  # This workflow contains a single job called "build"
  build:
    # The type of runner that the job will run on
    runs-on: ubuntu-latest

    # Steps represent a sequence of tasks that will be executed as part of the job
    steps:
      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
      - name: ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️ Check out this repo ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️
        uses: actions/checkout@v3

      - run:
          echo $(pwd) &&
          ls -lah &&
          sleep infinity
          
      - name: 📦📦📦📦📦📦📦📦 If act install packages 📦📦📦📦📦📦📦📦
        if: ${{ env.ACT }}
        run: |
          apt-get update -y
          apt-get install git-core zstd -y

      - name: ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️ Clone gb-studio ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️
        run: |
          git clone https://orcutt989:${{ secrets.GH_TOKEN }}@github.com/orcutt989/gb-studio.git

      - name: 📂📂📂📂📂📂📂📂 Yarn cache 📂📂📂📂📂📂📂📂
        uses: c-hive/gha-yarn-cache@v2
        if: ${{ !env.ACT }}
        with:
          directory: gb-studio

      - name: 🧶🧶🧶🧶🧶🧶🧶🧶 GitHub Action for Yarn 🧶🧶🧶🧶🧶🧶🧶🧶
        uses: Borales/actions-yarn@v3.0.0
        with:
          cmd: --cwd gb-studio install # will run `yarn install` command

      # Setup Node
      - name: 🛠️🛠️🛠️🛠️🛠️🛠️🛠️🛠️ Setup Node 🛠️🛠️🛠️🛠️🛠️🛠️🛠️🛠️
        uses: actions/setup-node@v3
        with:
          node-version: lts/*
          
      - name: 🏗️🏗️🏗️🏗️🏗️🏗️🏗️🏗️ Build gb-studio 🏗️🏗️🏗️🏗️🏗️🏗️🏗️🏗️
        run: |
          npm run make:cli
        working-directory: gb-studio

      - uses: Borales/actions-yarn@v3.0.0
        with:
          cmd: set version 1.22.4

      - name: 🔗🔗🔗🔗🔗🔗🔗🔗 Link GB Studio CLI binary 🔗🔗🔗🔗🔗🔗🔗🔗
        uses: Borales/actions-yarn@v3.0.0
        with:
          cmd: --cwd gb-studio link

      - name: 📋📋📋📋📋📋📋📋 GB Studio install check 📋📋📋📋📋📋📋📋
        run: |
          echo "********************************"
          sudo chmod a+x out/cli/gb-studio-cli.js
          VERSION=$(out/cli/gb-studio-cli.js -V)
          echo "🟨🟨🟨🟨🟨🟨🟨 GB-STUDIO-CLI VERSION: $VERSION 🟨🟨🟨🟨🟨🟨🟨🟨"
          echo "********************************"
        working-directory: gb-studio


      - name: 🎮🎮🎮🎮🎮🎮🎮🎮 build game 🎮🎮🎮🎮🎮🎮🎮🎮
        run: |
          gb-studio/out/cli/gb-studio-cli.js make:web paris-syndrome.gbsproj web/

      - name: 🚀🚀🚀🚀🚀🚀🚀🚀 Publish web game to Itch.io 🚀🚀🚀🚀🚀🚀🚀🚀
        uses: KikimoraGames/itch-publish@v0.0.3
        with:
          butlerApiKey: ${{secrets.BUTLER_API_KEY}}
          gameData: ./web
          itchUsername: ${{env.ITCH_USERNAME}}
          itchGameId: ${{ env.ITCH_GAME_ID }}
          buildChannel: webgl
```

### Relevant log output

```sh
PS C:\Users\xxx\src\paris-syndrome-gb> act -v
time="2022-06-30T13:36:44-04:00" level=debug msg="Loading environment from C:\\Users\\xxx\\src\\paris-syndrome-gb\\.env"
time="2022-06-30T13:36:44-04:00" level=debug msg="Loading secrets from C:\\Users\\xxx\\src\\paris-syndrome-gb\\my.secrets"
time="2022-06-30T13:36:44-04:00" level=debug msg="Loading workflows from 'C:\\Users\\xxx\\src\\paris-syndrome-gb\\.github\\workflows'"
time="2022-06-30T13:36:44-04:00" level=debug msg="Loading workflows recursively"
time="2022-06-30T13:36:44-04:00" level=debug msg="Found workflow 'publish-to-itch.yml' in 'C:\\Users\\xxx\\src\\paris-syndrome-gb\\.github\\workflows\\publish-to-itch.yml'"
time="2022-06-30T13:36:44-04:00" level=debug msg="Reading workflow 'C:\\Users\\xxx\\src\\paris-syndrome-gb\\.github\\workflows\\publish-to-itch.yml'"
time="2022-06-30T13:36:44-04:00" level=debug msg="Planning event: push"
time="2022-06-30T13:36:44-04:00" level=debug msg="Loading slug from git directory 'C:\\Users\\xxx\\src\\paris-syndrome-gb\\.git'"
time="2022-06-30T13:36:44-04:00" level=debug msg="Loading revision from git directory 'C:\\Users\\xxx\\src\\paris-syndrome-gb\\.git'"
time="2022-06-30T13:36:44-04:00" level=debug msg="Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f\n"
time="2022-06-30T13:36:44-04:00" level=debug msg="HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'"
time="2022-06-30T13:36:44-04:00" level=debug msg="using github ref: refs/heads/master"
time="2022-06-30T13:36:44-04:00" level=debug msg="Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f\n"
time="2022-06-30T13:36:44-04:00" level=debug msg="evaluating expression 'true'"
time="2022-06-30T13:36:44-04:00" level=debug msg="expression 'true' evaluated to 'true'"
time="2022-06-30T13:36:44-04:00" level=debug msg="evaluating expression 'honestpizza'"
time="2022-06-30T13:36:44-04:00" level=debug msg="expression 'honestpizza' evaluated to '%!t(<nil>)'"
time="2022-06-30T13:36:44-04:00" level=debug msg="evaluating expression 'paris-syndrome'"
time="2022-06-30T13:36:44-04:00" level=debug msg="expression 'paris-syndrome' evaluated to '%!t(<nil>)'"
[CI/build] [DEBUG] evaluating expression 'success()'
[CI/build] [DEBUG] expression 'success()' evaluated to 'true'
[CI/build] 🚀  Start image=ghcr.io/catthehacker/ubuntu:act-latest
[CI/build]   🐳  docker pull image=ghcr.io/catthehacker/ubuntu:act-latest platform= username= forcePull=false
[CI/build] [DEBUG]   🐳  docker pull ghcr.io/catthehacker/ubuntu:act-latest
[CI/build] [DEBUG] Image exists? true
[CI/build] [DEBUG]   🐳  docker volume rm act-CI-build
[CI/build] [DEBUG]   🐳  docker volume rm act-CI-build-env
[CI/build]   🐳  docker create image=ghcr.io/catthehacker/ubuntu:act-latest platform= entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[CI/build] [DEBUG] Created container name=act-CI-build id=e4ae4dd0c729feb37e92e68074641c943997d9305865d249510981a9276fa853 from image ghcr.io/catthehacker/ubuntu:act-latest (platform: )
[CI/build] [DEBUG] ENV ==> [RUNNER_TOOL_CACHE=/opt/hostedtoolcache RUNNER_OS=Linux RUNNER_TEMP=/tmp]
[CI/build]   🐳  docker run image=ghcr.io/catthehacker/ubuntu:act-latest platform= entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[CI/build] [DEBUG] Starting container: e4ae4dd0c729feb37e92e68074641c943997d9305865d249510981a9276fa853
[CI/build] [DEBUG] Started container: e4ae4dd0c729feb37e92e68074641c943997d9305865d249510981a9276fa853
[CI/build]   🐳  docker exec cmd=[chown -R 0:0 /mnt/c/Users/xxx/src/paris-syndrome-gb] user=0 workdir=
[CI/build] [DEBUG] Exec command '[chown -R 0:0 /mnt/c/Users/xxx/src/paris-syndrome-gb]'
[CI/build] [DEBUG] Working directory '/mnt/c/Users/xxx/src/paris-syndrome-gb'
[CI/build] [DEBUG] Writing entry to tarball workflow/event.json len:2
[CI/build] [DEBUG] Writing entry to tarball workflow/envs.txt len:0
[CI/build] [DEBUG] Writing entry to tarball workflow/paths.txt len:0
[CI/build] [DEBUG] Extracting content to '/var/run/act/'
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] Skipping local actions/checkout because workdir was already copied
[CI/build] [DEBUG] skip pre step for '⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️ Check out this repo ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️': no action model available
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build]   ☁  git clone 'https://github.com/c-hive/gha-yarn-cache' # ref=v2
[CI/build] [DEBUG]   cloning https://github.com/c-hive/gha-yarn-cache to C:\Users\xxx\.cache\act/c-hive-gha-yarn-cache@v2
[CI/build] [DEBUG] Cloned https://github.com/c-hive/gha-yarn-cache to C:\Users\xxx\.cache\act/c-hive-gha-yarn-cache@v2
[CI/build] [DEBUG] Checked out v2
[CI/build] [DEBUG] Read action &{Yarn install cache c-hive 1-liner yarn install cache for GitHub Actions map[directory:{Directory containing yarn.lock false .}] map[cache-hit:{A boolean value to indicate an exact match was found for the primary key }] {node12 map[] dist/restore/index.js  always() dist/save/index.js success()   [] []} { }} from 'Unknown'
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build]   ☁  git clone 'https://github.com/Borales/actions-yarn' # ref=v3.0.0
[CI/build] [DEBUG]   cloning https://github.com/Borales/actions-yarn to C:\Users\xxx\.cache\act/Borales-actions-yarn@v3.0.0
[CI/build] [DEBUG] Cloned https://github.com/Borales/actions-yarn to C:\Users\xxx\.cache\act/Borales-actions-yarn@v3.0.0
[CI/build] [DEBUG] Checked out v3.0.0
[CI/build] [DEBUG] Read action &{GitHub Action for Yarn  Wraps the yarn CLI to enable common yarn commands map[auth-token:{NPM_AUTH_TOKEN false } cmd:{Yarn command true } registry-url:{NPM_REGISTRY_URL false }] map[] {docker map[NPM_AUTH_TOKEN:${{ inputs.auth-token }} NPM_REGISTRY_URL:${{ inputs.registry-url }}]   always()  always() Dockerfile  [${{ inputs.cmd }}] []} {blue package}} from 'Unknown'
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build]   ☁  git clone 'https://github.com/actions/setup-node' # ref=v3
[CI/build] [DEBUG]   cloning https://github.com/actions/setup-node to C:\Users\xxx\.cache\act/actions-setup-node@v3
[CI/build] [DEBUG] Cloned https://github.com/actions/setup-node to C:\Users\xxx\.cache\act/actions-setup-node@v3
[CI/build] [DEBUG] Checked out v3
[CI/build] [DEBUG] Read action &{Setup Node.js environment GitHub Setup a Node.js environment by adding problem matchers and optionally downloading and adding it to the PATH. map[always-auth:{Set always-auth in npmrc. false false} architecture:{Target architecture for Node to use. Examples: x86, x64. Will use system architecture by default. false } cache:{Used to specify a package manager for caching in the default directory. Supported values: npm, yarn, pnpm. false } cache-dependency-path:{Used to specify the path to a dependency file: package-lock.json, yarn.lock, etc. Supports wildcards or a list of file names for caching multiple dependencies. false } check-latest:{Set this option if you want the action to check for the latest available version that satisfies the version spec. false false} node-version:{Version Spec of the version to use. Examples: 12.x, 10.15.1, >=10.15.0. false } node-version-file:{File containing the version Spec of the version to use.  Examples: .nvmrc, .node-version. false } registry-url:{Optional registry to set up for auth. Will set the registry in a project level .npmrc and .yarnrc file, and set up auth to read in from env.NODE_AUTH_TOKEN. false } scope:{Optional scope for authenticating against scoped registries. Will fall back to the repository owner when using the GitHub Packages registry (https://npm.pkg.github.com/). false } token:{Used to pull node distributions from node-versions.  Since there's a default, this is typically not supplied by the user. false ${{ github.token }}}] map[cache-hit:{A boolean value to indicate if a cache was hit. }] {node16 map[] dist/setup/index.js  always() dist/cache-save/index.js success()   [] []} { }} from 'Unknown'
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] expression '${{ github.token }}' rewritten to 'format('{0}', github.token)'
[CI/build] [DEBUG] evaluating expression 'format('{0}', github.token)'
[CI/build] [DEBUG] expression 'format('{0}', github.token)' evaluated to '%!t(string=***)'
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build]   ☁  git clone 'https://github.com/Borales/actions-yarn' # ref=v3.0.0
[CI/build] [DEBUG]   cloning https://github.com/Borales/actions-yarn to C:\Users\xxx\.cache\act/Borales-actions-yarn@v3.0.0
[CI/build] [DEBUG] Cloned https://github.com/Borales/actions-yarn to C:\Users\xxx\.cache\act/Borales-actions-yarn@v3.0.0
[CI/build] [DEBUG] Checked out v3.0.0
[CI/build] [DEBUG] Read action &{GitHub Action for Yarn  Wraps the yarn CLI to enable common yarn commands map[auth-token:{NPM_AUTH_TOKEN false } cmd:{Yarn command true } registry-url:{NPM_REGISTRY_URL false }] map[] {docker map[NPM_AUTH_TOKEN:${{ inputs.auth-token }} NPM_REGISTRY_URL:${{ inputs.registry-url }}]   always()  always() Dockerfile  [${{ inputs.cmd }}] []} {blue package}} from 'Unknown'
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build]   ☁  git clone 'https://github.com/Borales/actions-yarn' # ref=v3.0.0
[CI/build] [DEBUG]   cloning https://github.com/Borales/actions-yarn to C:\Users\xxx\.cache\act/Borales-actions-yarn@v3.0.0
[CI/build] [DEBUG] Cloned https://github.com/Borales/actions-yarn to C:\Users\xxx\.cache\act/Borales-actions-yarn@v3.0.0
[CI/build] [DEBUG] Checked out v3.0.0
[CI/build] [DEBUG] Read action &{GitHub Action for Yarn  Wraps the yarn CLI to enable common yarn commands map[auth-token:{NPM_AUTH_TOKEN false } cmd:{Yarn command true } registry-url:{NPM_REGISTRY_URL false }] map[] {docker map[NPM_AUTH_TOKEN:${{ inputs.auth-token }} NPM_REGISTRY_URL:${{ inputs.registry-url }}]   always()  always() Dockerfile  [${{ inputs.cmd }}] []} {blue package}} from 'Unknown'
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build]   ☁  git clone 'https://github.com/KikimoraGames/itch-publish' # ref=v0.0.3
[CI/build] [DEBUG]   cloning https://github.com/KikimoraGames/itch-publish to C:\Users\xxx\.cache\act/KikimoraGames-itch-publish@v0.0.3
[CI/build] [DEBUG] Cloned https://github.com/KikimoraGames/itch-publish to C:\Users\xxx\.cache\act/KikimoraGames-itch-publish@v0.0.3
[CI/build] [DEBUG] Checked out v0.0.3
[CI/build] [DEBUG] Read action &{Itch.io - Publish KikimoraGames Publish releases to Itch.io map[buildChannel:{Channel name of the game: https://itch.io/docs/butler/pushing.html#channel-names true } buildNumber:{Optional build number, use to supply your own build version instead of using itch's versioning false } buildNumberFile:{Optional build number file path, use to supply your own build version instead of using itch's versioning. The file should contain a single line with the version or build number, in UTF-8 without BOM. false } butlerApiKey:{Butler API Key true } gameData:{Directory or .zip file of the game data. Zip files are slower to upload. true } itchGameId:{Itch.io id of the game. e.g. in finji/overland this would be overland. true } itchUsername:{Itch.io username of the game owner. e.g. in finji/overland this would be finji. true }] map[] {docker map[]   always()  always() Dockerfile  [] []} {red upload-cloud}} from 'Unknown'
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] expression '${{env.ITCH_USERNAME}}' rewritten to 'format('{0}', env.ITCH_USERNAME)'
[CI/build] [DEBUG] evaluating expression 'format('{0}', env.ITCH_USERNAME)'
[CI/build] [DEBUG] expression 'format('{0}', env.ITCH_USERNAME)' evaluated to '%!t(string=honestpizza)'
[CI/build] [DEBUG] expression '${{ env.ITCH_GAME_ID }}' rewritten to 'format('{0}', env.ITCH_GAME_ID)'
[CI/build] [DEBUG] evaluating expression 'format('{0}', env.ITCH_GAME_ID)'
[CI/build] [DEBUG] expression 'format('{0}', env.ITCH_GAME_ID)' evaluated to '%!t(string=paris-syndrome)'
[CI/build] [DEBUG] expression '${{secrets.BUTLER_API_KEY}}' rewritten to 'format('{0}', secrets.BUTLER_API_KEY)'
[CI/build] [DEBUG] evaluating expression 'format('{0}', secrets.BUTLER_API_KEY)'
[CI/build] [DEBUG] expression 'format('{0}', secrets.BUTLER_API_KEY)' evaluated to '%!t(string=***)'
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] Skipping local actions/checkout because workdir was already copied
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] setupEnv => map[ACT:true AGENT_TOOLSDIRECTORY:/opt/hostedtoolcache CI:true DEBIAN_FRONTEND:noninteractive DEPLOYMENT_BASEPATH:/opt/runner GITHUB_ACTION:0 GITHUB_ACTIONS:true GITHUB_ACTION_PATH: GITHUB_ACTION_REF: GITHUB_ACTION_REPOSITORY: GITHUB_ACTOR:nektos/act GITHUB_API_URL:https://api.github.com GITHUB_BASE_REF: GITHUB_ENV:/var/run/act/workflow/envs.txt GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/var/run/act/workflow/event.json GITHUB_GRAPHQL_URL:https://api.github.com/graphql GITHUB_HEAD_REF: GITHUB_JOB:build GITHUB_PATH:/var/run/act/workflow/paths.txt GITHUB_REF:refs/heads/master GITHUB_REF_NAME:master GITHUB_REF_TYPE:branch GITHUB_REPOSITORY:orcutt989/paris-syndrome-gb GITHUB_REPOSITORY_OWNER:orcutt989 GITHUB_RETENTION_DAYS:0 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SERVER_URL:https://github.com GITHUB_SHA:ade85c8271b4d45b3bb3d276e473e8bc5d88a92f GITHUB_TOKEN:*** GITHUB_WORKFLOW:CI GITHUB_WORKSPACE:/mnt/c/Users/xxx/src/paris-syndrome-gb IMAGE_OS:ubuntu20 ITCH_GAME_ID:paris-syndrome ITCH_USERNAME:honestpizza ImageOS:ubuntu20 LSB_OS_VERSION:2004 LSB_RELEASE:20.04 PATH:/opt/hostedtoolcache/node/16.15.0/x64/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin RUNNER_PERFLOG:/dev/null RUNNER_TRACKING_ID: RUNNER_USER:root RUN_TOOL_CACHE:/opt/hostedtoolcache USER:root]
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] evaluating expression ''
[CI/build] [DEBUG] expression '' evaluated to 'true'
[CI/build] ⭐ Run Main ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️ Check out this repo ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build]   🐳  docker cp src=C:\Users\xxx\src\paris-syndrome-gb\. dst=\mnt\c\Users\xxx\src\paris-syndrome-gb
[CI/build] [DEBUG] Writing tarball C:\Users\xxx\AppData\Local\Temp\act3763354995 from C:\Users\xxx\src\paris-syndrome-gb\.
[CI/build] [DEBUG] Stripping prefix:C:\Users\xxx\src\paris-syndrome-gb\ src:C:\Users\xxx\src\paris-syndrome-gb\.
[CI/build] [DEBUG] Extracting content from 'C:\Users\xxx\AppData\Local\Temp\act3763354995' to '\mnt\c\Users\xxx\src\paris-syndrome-gb'
[CI/build] close C:\Users\xxx\AppData\Local\Temp\act3763354995: file already closed
[CI/build]   🐳  docker exec cmd=[chown -R 0:0 \mnt\c\Users\xxx\src\paris-syndrome-gb] user=0 workdir=
[CI/build] [DEBUG] Exec command '[chown -R 0:0 /mnt/c/Users/xxx/src/paris-syndrome-gb]'
[CI/build] [DEBUG] Working directory '/mnt/c/Users/xxx/src/paris-syndrome-gb'
[CI/build]   ✅  Success - Main ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️ Check out this repo ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] setupEnv => map[ACT:true AGENT_TOOLSDIRECTORY:/opt/hostedtoolcache CI:true DEBIAN_FRONTEND:noninteractive DEPLOYMENT_BASEPATH:/opt/runner GITHUB_ACTION:1 GITHUB_ACTIONS:true GITHUB_ACTION_PATH: GITHUB_ACTION_REF: GITHUB_ACTION_REPOSITORY: GITHUB_ACTOR:nektos/act GITHUB_API_URL:https://api.github.com GITHUB_BASE_REF: GITHUB_ENV:/var/run/act/workflow/envs.txt GITHUB_EVENT_NAME:push GITHUB_EVENT_PATH:/var/run/act/workflow/event.json GITHUB_GRAPHQL_URL:https://api.github.com/graphql GITHUB_HEAD_REF: GITHUB_JOB:build GITHUB_PATH:/var/run/act/workflow/paths.txt GITHUB_REF:refs/heads/master GITHUB_REF_NAME:master GITHUB_REF_TYPE:branch GITHUB_REPOSITORY:orcutt989/paris-syndrome-gb GITHUB_REPOSITORY_OWNER:orcutt989 GITHUB_RETENTION_DAYS:0 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SERVER_URL:https://github.com GITHUB_SHA:ade85c8271b4d45b3bb3d276e473e8bc5d88a92f GITHUB_TOKEN:*** GITHUB_WORKFLOW:CI GITHUB_WORKSPACE:/mnt/c/Users/xxx/src/paris-syndrome-gb IMAGE_OS:ubuntu20 ITCH_GAME_ID:paris-syndrome ITCH_USERNAME:honestpizza ImageOS:ubuntu20 LSB_OS_VERSION:2004 LSB_RELEASE:20.04 PATH:/opt/hostedtoolcache/node/16.15.0/x64/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin RUNNER_PERFLOG:/dev/null RUNNER_TRACKING_ID: RUNNER_USER:root RUN_TOOL_CACHE:/opt/hostedtoolcache USER:root]
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] evaluating expression ''
[CI/build] [DEBUG] expression '' evaluated to 'true'
[CI/build] ⭐ Run Main echo $(pwd) && ls -lah && sleep infinity
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] Loading slug from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Loading revision from git directory 'C:\Users\xxx\src\paris-syndrome-gb\.git'
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] HEAD points to 'ade85c8271b4d45b3bb3d276e473e8bc5d88a92f'
[CI/build] [DEBUG] using github ref: refs/heads/master
[CI/build] [DEBUG] Found revision: ade85c8271b4d45b3bb3d276e473e8bc5d88a92f
[CI/build] [DEBUG] Wrote command

echo $(pwd) && ls -lah && sleep infinity

 to 'workflow/1'
[CI/build] [DEBUG] Writing entry to tarball workflow/1 len:42
[CI/build] [DEBUG] Extracting content to '/var/run/act'
[CI/build]   🐳  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/1] user= workdir=
[CI/build] [DEBUG] Exec command '[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/1]'
[CI/build] [DEBUG] Working directory '/mnt/c/Users/xxx/src/paris-syndrome-gb'
| /mnt/c/Users/xxx/src/paris-syndrome-gb
| total 8.0K
| drwxr-xr-x 2 root root 4.0K Jun 30 17:36 .
| drwxr-xr-x 3 root root 4.0K Jun 30 17:36 ..
```

### Additional information

Files are not in `.gitignore`.

I assume I am doing something incorrectly as opposed to this being a bug in full disclosure. 😄

**Comment by ChristopherHX** [comment]
Duplicate of #1227
It's a bug of act 0.2.27 and 0.2.28 on windows and is already fixed in master.

**Comment by orcutt989** [comment]
Thanks how to build on windows? When I run `make test` I get a buch of failures and when I `make install` I get this error. 

```
cp: cannot create regular file '/usr/local/bin/act': No such file or directory
```

**Comment by ChristopherHX** [comment]
If you have go 1.18 installed and in `PATH`

```
go build
```
This creates a `act.exe` file, which is ready to use. I never used the makefile, it seem to contain linux / mac paths.

Otherwise 0.2.29 should be released in a few hours, by an automated github workflow.

**Comment by orcutt989** [comment]
That works and gets me cloning, but what's funny is it seems emojis stop working when built locally? Happy to wait until the CI pipeline runs, I just thought it was an interesting.
